libs/parallelism/algorithms/include/hpx/parallel/algorithms/transform_reduce.hpp

```diff
@@ -326,25 +388,20 @@ namespace hpx { namespace parallel { inline namespace v1 {
                 using reference =
                     typename std::iterator_traits<Iter>::reference;
```

Comment:
```suggestion
```
Would you mind removing this typedef? It's now unused (see https://app.circleci.com/pipelines/github/STEllAR-GROUP/hpx/7241/workflows/7d3d3531-1e85-4225-bc4e-b46c0ef3bd4f/jobs/245396/steps).

Reply:
Sure, there's actually an unused one in the sequential version as well.